AdvanceSpecFlowProject/SpecFlowProject/Steps/ShareSkillSteps/ManageListingsComponentSteps.cs

```diff
@@ -0,0 +1,252 @@
+                string expectedMessage = "Service has been activated";
+                string actualMessage = ManageListingsComponentObj.GetMessage();
+                Assert.AreEqual(expectedMessage, actualMessage, "Actual and expected share skill do not match. Share skill not activated!!");
+            }
+            catch (Exception ex)
+            {
+                Assert.Fail();
+            }
+        }
+        public void DeactivateShareSkill(SkillModel skill)
+        {
+            //--------Deavtivate a skill--------
+            try
+            {
+                ManageListingsComponentObj.ChangeShareSkillStatus(skill);
+                //Verify the message displayed on deactivating share skill
+                string expectedMessage = "Service has been deactivated";
+                string actualMessage = ManageListingsComponentObj.GetMessage();
+                Assert.AreEqual(expectedMessage, actualMessage, "Actual and expected share skill do not match. Share skill not deactivated!!");
+            }
+            catch (Exception ex)
+            {
+                Assert.Fail();
+            }
+        }
+        public void NavigateToNextPage()
+        {
+            //--------Navigate to next page----
+            try
+            {
+                int expectedNextPage = ManageListingsComponentObj.GetActivePageNumber() + 1;
+                ManageListingsComponentObj.NavigateToNextPage();
+                int actualNextPage = ManageListingsComponentObj.GetActivePageNumber();
+                Assert.AreEqual(expectedNextPage, actualNextPage, "Actual and expected page number do not match. Not navigated to next page!!");
+            }
+            catch (Exception ex)
+            {
+                Assert.Fail();
+            }
+        }
+        public void NavigateToPreviousPage()
+        {
+            //--------Navigate to previous page --------
+            try
+            {
+                ManageListingsComponentObj.NavigateToNextPage();
+                int expectedPrevioustPage = ManageListingsComponentObj.GetActivePageNumber() - 1;
+                ManageListingsComponentObj.NavigateToPreviousPage();
+                int actualPreviousPage = ManageListingsComponentObj.GetActivePageNumber();
+                Assert.AreEqual(expectedPrevioustPage, actualPreviousPage, "Actual and expected page number do not match. Not navigated to previous page!!");
+            }
+            catch (Exception ex)
+            {
+                Assert.Fail();
+            }
+        }
+        public void ValidateShareSkillAdded(SkillModel skill)
+        {
+            //--------Validate askill is added--------
+            try
+            {
+                //Verify the added share skill record in the table
+                string additionStatus = ManageListingsComponentObj.GetShareSkillAddedStatus(skill);
+                Assert.AreEqual("Added", additionStatus, "Actual and expected do not match. Share skill not added!!");
```

Comment:
I would move all assertions in to another class, e.g SkillsAssertionHelper its cleaner to keep all assertions related to features in one place please fix this

Reply:
Issue resolved. Please review.